SConstruct

```diff
@@ -7,6 +7,7 @@ import SCons.Script
 # supported platform names
 supported_platforms = [
     'linux',
+    'freebsd',
```

Comment:
What do you think about other BSDs? If in future we will also add support, say, for OpenBSD, will we need a different platform for it? Or maybe we can have a single `bsd` platform for all of them, instead of `freebsd`, `openbsd`, etc? If you don't known, we can keep `freebsd` for now.

Reply:
I don't know what kinds of problems are awaiting on those other problems.
Sure, having just a single `bsd` platform sounds nice, but until this is shown to actually work I'd stick with the more specific ones.

Reply:
OK.